db/migrate/20171214143221_create_comments.rb

```diff
@@ -1,5 +1,6 @@
 class CreateComments < ActiveRecord::Migration[5.1]
   def change
+    drop_table :comments
```

Comment:
Hey @madoleary I forgot to ask, why did you put that there ?

Reply:
db:migrate wasn't working for me with an error occurring with respect to comments so i dropped the table before completing the migration.

Reply:
Ok, are you ok if I comment that out in production ? If it doesn't work, well... we'll fix it ahaha

Reply:
sure, should have commented it out before pushing, so my bad.

Reply:
okidokie